scripts/deploySidechain.ts

```diff
@@ -663,3 +665,41 @@ export async function setTrustedRemoteCanonicalPhase2(
     tx = await canonical.auraBalProxyOFT.transferOwnership(multisigs.daoMultisig);
     await waitForTx(tx, debug, waitForBlocks);
 }
+
+export async function deploySidechainClaimZap(
+    extConfig: ExtSidechainConfig,
+    sidechain: SidechainPhase1Deployed & SidechainPhase2Deployed,
+    hre: HardhatRuntimeEnvironment,
+    signer: Signer,
+    salt: string = SALT,
+    debug = false,
+    waitForBlocks = 0,
+): Promise<{ sidechainClaimZap: SidechainClaimZap }> {
+    const create2Options = { amount: 0, salt, callbacks: [] };
+    const deployOptions = {
+        overrides: {},
+        create2Options,
+        debug,
+        waitForBlocks,
+    };
+
+    const create2Factory = Create2Factory__factory.connect(extConfig.create2Factory, signer);
+
+    const sidechainClaimZap = await deployContractWithCreate2<SidechainClaimZap, SidechainClaimZap__factory>(
+        hre,
+        create2Factory,
+        new SidechainClaimZap__factory(signer),
+        "SidechainClaimZap",
+        [],
+        deployOptions,
+    );
+
+    const tx = await sidechainClaimZap.init(
```

Comment:
Send this in as a part of the create2 options rather than calling it second, just in case some griefer really wants to frontrun this 😂 Then can pass in `owner` as a param rather than using `msg.sender` in the init function.